src/config/services/servicesConfig.go

```diff
@@ -107,6 +116,14 @@ func DecodeConfig(cfg config.Provider) (c Config, err error) {
 		c.RSS.Limit = cfg.GetInt(rssLimitKey)
 	}
 
+	if c.JSONFeed.Limit == 0 {
```

Comment:
When would this be true? Is it not based on what the default. Same for line 123.

Reply:
The majority of the time. An integer's default value is '0' so when this value is not assigned, this will be the value. Same for line 123.